Src/BootCamp.Chapter/Program.cs

```diff
@@ -7,6 +7,26 @@ class Program
         static void Main(string[] args)
         {
             Console.WriteLine("Hello World!");
-        }
-    }
+			{//Line that will ask user for their name.
+				Console.Write("Input your full name: ");
+				string name = Console.ReadLine();
+				Console.WriteLine("Hello again, " + name);
+				
+				//Line that will ask for age. 
+				Console.Write(name + ", what is your age? ");
+				string age = Console.ReadLine();
+
+				//Line that reads users weight and height.
+				Console.Write(name + ", what is your weight in kgs? ");
+				string weight = Console.ReadLine();
+				Console.Write(name+", what is your height in cm? ");
+				string height = Console.ReadLine();
+
+				Console.WriteLine(name + " is " + age + " years old, his weight is " + weight +
+					" kgs and his height is " + height + " cm.");
+			}
+
+		}
+
```

Comment:
You have to change your code to be this:
```cs
static void Main(string[] args)
{
	{
		Console.Write("Input your full name: ");
		string name = Console.ReadLine();
		Console.WriteLine("Hello again, " + name);

		//Line that will ask for age. 
		Console.Write(name + ", what is your age? ");
		string age = Console.ReadLine();

		//Line that reads users weight and height.
		Console.Write(name + ", what is your weight in kgs? ");
		string weight = Console.ReadLine();
		Console.Write(name+", what is your height in cm? ");
		string height = Console.ReadLine();

		Console.WriteLine(name + " is " + age + " years old, his weight is " + weight +
				" kgs and his height is " + height + " cm.");
	}
}
```
After that you have to request the data for another person.
But please make the change above first, then commit, then push, then do for another person. :)